databuilder/query_engines/sqlite.py

```diff
@@ -0,0 +1,178 @@
+import contextlib
+from functools import cache, cached_property, singledispatchmethod
+
+import sqlalchemy
+from sqlalchemy.dialects.sqlite.pysqlite import SQLiteDialect_pysqlite
+
+from databuilder.query_model import Function, SelectColumn, SelectPatientTable, Value
+from databuilder.sqlalchemy_utils import get_referenced_tables
+
+from .base import BaseQueryEngine
+
+
+class SQLiteQueryEngine(BaseQueryEngine):
+
+    sqlalchemy_dialect = SQLiteDialect_pysqlite
+
+    def get_query(self, column_definitions):
```

Comment:
Can we switch to "variable" instead of "column" for consistency with terminology elsewhere?